sdk/storage/azure-storage-blob/tests/test_common_blob_async.py

```diff
@@ -434,10 +434,12 @@ def test_get_blob_with_snapshot_previous(self):
         content = await stream.content_as_bytes()
 
         # Assert
-        self.assertEqual(content, self.byte_data[:6])
+        self.assertEqual(content, self.byte_data[:5])
 
     @record
     def test_get_blob_with_range(self):
+        if TestMode.need_recording_file(self.test_mode):
+            pytest.skip("Issue with the recording")
```

Comment:
Do you know what caused the problem for recording? ^_^

Reply:
recording doesn't add the `content-type` for some reason.

Reply:
Did we create an item for fixing that 😀

Reply:
Added to the AC here: https://github.com/Azure/azure-sdk-for-python/issues/6225